velox/exec/tests/SimpleArrayAggAggregate.cpp

```diff
@@ -148,7 +148,8 @@ exec::AggregateRegistrationResult registerSimpleArrayAggAggregate(
             argTypes.size(), 1, "{} takes at most one argument", name);
         return std::make_unique<SimpleAggregateAdapter<ArrayAggAggregate>>(
             resultType);
-      });
+      },
+      /*registerCompanionFunctions*/ true);
```

Comment:
```
/*registerCompanionFunctions=*/
```

Reply:
I'd prefer to not have "=" in the comment to be consistent with other places in our codebase. E.g., https://github.com/facebookincubator/velox/blob/d2ebb4e3ba432882d93065eff94d9dad4101bb8b/velox/dwio/dwrf/reader/ColumnReader.cpp#L364-L369